examples/rpc/partner-plugin/services/cache/cache.go

```diff
@@ -0,0 +1,196 @@
+package cache
+
+import (
+	"sync"
+	"time"
+
+	accommodationv1 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/services/accommodation/v1"
+	accommodationv2 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/services/accommodation/v2"
+	typesv1 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/types/v1"
+	typesv2 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/types/v2"
+)
+
+type searchCache struct {
+	mu       sync.RWMutex
+	cache_v1 map[string]*cachedResultV1
+	cache_v2 map[string]*cachedResultV2
+}
+
+type cachedResultV1 struct {
```

Comment:
The caching implementation seems pretty hard-wired to accommodation search results. Maybe there is a way to generalize this to be able to support more services?
Or in general I'd raise the question if caching is actually needed for static mock-data.

Reply:
cache service removed.